libs/nx-react-workshop/src/migrations/complete-lab-1/complete-lab-1.ts

```diff
@@ -11,39 +11,13 @@ import { removeGenerator } from '@nx/workspace';
 import { execSync } from 'child_process';
 
 export default async function update(tree: Tree) {
-  // npx create-nx-workspace bg-hoard --preset=empty --no-nx-cloud
-  const projects = getProjects(tree);
-  const projectsToRemove = [
-    'store-e2e',
-    'store',
-    'api',
-    'api-e2e',
-    'api-util-interface',
-    'util-interface',
-    'store-feature-game-detail',
-    'ui-shared',
-    'store-ui-shared',
-    'store-ui-shared-e2e',
-    'store-util-formatters',
-    'api-util-notifications',
-    'admin-ui',
-    'admin-ui-e2e',
-    'internal-plugin',
-    'internal-plugin-e2e',
-  ].filter((removeProject) => projects.has(removeProject));
-  projectsToRemove.forEach(
-    async (projectName) =>
-      await removeGenerator(tree, {
-        projectName,
-        skipFormat: true,
-        forceRemove: true,
-      })
-  );
-  // hack to fix remove generator
-  updateJson(tree, 'tsconfig.base.json', (json) => {
-    json.compilerOptions.paths = {};
-    return json;
-  });
+  for (const [projectName] of getProjects(tree)) {
+    await removeGenerator(tree, {
+      projectName,
+      skipFormat: true,
+      forceRemove: true,
+    });
+  }
```

Comment:
Since we pass `forceRemove`, we don't need to enumerate the projects and delete them in a specific order.